src/server/snapshot.cc

```diff
@@ -105,16 +107,18 @@ void SliceSnapshot::Start(bool stream_journal, SnapshotFlush allow_flush) {
 
   VLOG(1) << "DbSaver::Start - saving entries with version less than " << snapshot_version_;
 
-  string fb_name = absl::StrCat("SliceSnapshot-", ProactorBase::me()->GetPoolIndex());
-  snapshot_fb_ = fb2::Fiber(fb_name, [this, stream_journal] {
-    this->IterateBucketsFb(stream_journal);
-    db_slice_->UnregisterOnChange(snapshot_version_);
-    if (!use_snapshot_version_) {
-      db_slice_->UnregisterOnMoved(moved_cb_id_);
-    }
-    consumer_->Finalize();
-    VLOG(1) << "Serialization peak bytes: " << serializer_->GetSerializationPeakBytes();
-  });
+  snapshot_fb_name_ = absl::StrCat("SliceSnapshot-", ProactorBase::me()->GetPoolIndex());
+  auto prio = use_background_mode_ ? fb2::FiberPriority::BACKGROUND : fb2::FiberPriority::NORMAL;
+  snapshot_fb_ = fb2::Fiber(
+      fb2::Fiber::Opts{.priority = prio, .name = snapshot_fb_name_}, [this, stream_journal] {
```

Comment:
Fiber is created immediately, so it is ok for a string to be on stack as its name is copied during fiber creation. 
Having said that, I do think that .name should be converted to string to allow using StrCat without such complexities.